oaknational/Oak-Web-Application
# Feat/lesq 407/saved details [LESQ-407]

## Description

- Add a new 'edit' icon in sanity and use for the edit button
- Update the layout and content of `DetailsCompleted` component
- Move the `Your Details` header and use flex layout for school details section of the page

## How to test

1. Go to https://deploy-preview-2047--oak-web-application.netlify.thenational.academy/teachers/programmes/chemistry-secondary-ks4-l/units/atomic-structure-and-the-periodic-table-u68nqqc/lessons/atoms-elements-and-compounds-h3hasv/downloads
2. Fill in your details and make a download
3. You should see the details complete section matches the new designs

## Screenshots

How it used to look (delete if n/a):
<img width="490" alt="Screenshot 2023-10-25 at 09 43 52" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/a878dce0-14dc-4197-a45e-552ac84407bd">


How it should now look:
<img width="490" alt="Screenshot 2023-10-25 at 13 53 36" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/5df4d9bc-9fc0-4b26-927b-b1cd57f28f24">

## ACs
- [ ]  Add new edit icon
- [ ]  Add grey background shading
- [ ]  Update styles to match Figma
- [ ]  Copyright detail remains outside the box
- [ ]  When no email is provided it will say ‘Not provided’ under the email header
